modules/fb_instant_articles_display/fb_instant_articles_display.module

```diff
@@ -1,11 +1,34 @@
 <?php
-
```

Comment:
@m4olivei small style note, but let's add this extra linebreak back (I know Drupal core has some inconsistencies around this, but it's best practice to keep a space between the initial `<?php` and `@file` doxy block.